doc/mgr/nfs.rst

```diff
@@ -318,6 +318,11 @@ values may be separated by a comma (example: ``--sectype krb5p,krb5i``). The
 server will negotatiate a supported security type with the client preferring
 the supplied methods left-to-right.
 
+``<cmount_path>`` specifies the path within the ceph filesystem to mount this export on. It is
+allowed to be any complete path hierarchy between ``/`` and the ``EXPORT {path}``. (i.e. if ``EXPORT { Path }`` parameter is ``/foo/bar`` then cmount_path could be ``/``, ``/foo`` or ``/foo/bar``).
+If this and the other ``EXPORT { FSAL {} }`` options are the same between multiple exports, those exports will share a single cephfs client.
```

Comment:
!!

Reply:
> !!

Can u elaborate?

Reply:
i think he meant s/cephfs/CephFS

Reply:
@dparmar18  I did address that already!  I'm not sure about the `!!` in particular though